File: nodejs/aliexpress.js
const request = require('request');

const username = 'YOUR_USERNAME';
const password = 'YOUR_PASSWORD';

const options = {
  method: 'POST',
  url: 'https://scrape.smartproxy.com/v1/tasks',
  headers: {
    'Content-Type': 'application/json',
    'Authorization': 'Basic ' + Buffer.from(username + ':' + password).toString('base64')
  },
  body: JSON.stringify({
    target: 'aliexpress',
    parse: false,
    url: 'https://www.aliexpress.com/wholesale?&SearchText=louis+v'
  })
};

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});

File: python/aliexpress.py
import requests

task_params = {
    'target': 'aliexpress',
    'url': 'https://www.aliexpress.com/wholesale?&SearchText=louis+v',
    'parse': False,
}

username = 'SPusername'
password = 'SPpassword'

response = requests.post(
    'https://scrape.smartproxy.com/v1/tasks',
    json = task_params,
    auth = (username, password)
)

print(response.text)